services/workflows-service/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@ballerine/workflows-service",
   "private": false,
-  "version": "0.7.84",
+  "version": "0.7.85",
```

Comment:
You didn't change anything in the workflow-service, there's no reason to bump a version here. Please correct me if I'm wrong.